drivers/bbram/Kconfig.stm32

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022 Google Inc
```

Comment:
Just to verify and since provided mail differs: Is that accurate ?

Reply:
Yes